python/dgl/dataloading/base.py

```diff
@@ -279,13 +279,14 @@ def _find_exclude_eids_with_reverse_types(g, eids, reverse_etype_map):
         g.to_canonical_etype(k): g.to_canonical_etype(v)
         for k, v in reverse_etype_map.items()
     }
-    exclude_eids.update(
-        {
-            reverse_etype_map[k]: v
-            for k, v in exclude_eids.items()
-            if k in reverse_etype_map
-        }
-    )
+    for k, v in reverse_etype_map.items():
```

Comment:
I am not sure whether the _find_exclude_eids_with_reverse_types clearly states what inside the method.

Reply:
yes, it is exactly what the function do.